test/libsolidity/syntaxTests/inheritance/dataLocation/memory_calldata_internal.sol

```diff
@@ -0,0 +1,16 @@
+abstract contract A {
+    function f(uint256[1] memory a) internal virtual returns (uint256);
+    function test() external returns (uint) {
+        uint[1] memory t;
+        t[0] = 7;
+        return f(t);
+    }
+}
+
+contract B is A {
+    function f(uint256[1] calldata a) internal override returns (uint256) {
```

Comment:
Can you add an equivalent test for changing location in a modifier? We have it covered but not specifically in case of changing just the data location.